Library/Formula/llvm.rb

```diff
@@ -198,12 +196,6 @@ def install
 
     args << "-DLLVM_ENABLE_RTTI=On" if build.with? "rtti"
 
-    if build.with? "assertions"
-      args << "-DLLVM_ENABLE_ASSERTIONS=On"
-    else
-      args << "-DCMAKE_CXX_FLAGS_RELEASE='-DNDEBUG'"
```

Comment:
As far as I can tell the default Release flags are `-O3 -DNDEBUG`, so is adding `-DNDEBUG` to `std_cmake_args` an option? I don't know how many formulas use assertions in Release...